src/drivers/eeprom_i2c.c

```diff
@@ -0,0 +1,135 @@
+#include "simba.h"
+#if CONFIG_PUMBAA_CLASS_I2C_SOFT == 1
+struct i2c_soft_driver_t i2c;
+
+int eeprom_i2c_init(struct i2c_soft_driver_t *self_p,
+                  struct pin_device_t *scl_dev_p,
+                  struct pin_device_t *sda_dev_p,
+                  long baudrate,
+                  long max_clock_stretching_us,
+                  long clock_stretching_sleep_us)
+{
+    return i2c_soft_init(self_p,scl_dev_p,sda_dev_p,baudrate,max_clock_stretching_us,clock_stretching_sleep_us);
+}
+
+
+
+static int start_cond(struct i2c_soft_driver_t *self_p)
+{
+    return i2c_soft_start_cond(self_p);
+
+}
+
+
+static int stop_cond(struct i2c_soft_driver_t *self_p)
+{
+    return i2c_soft_stop_cond(self_p);
+
+}
+
+static int write_byte(struct i2c_soft_driver_t *self_p,
+                      uint8_t byte, int ACK)
+{
+    return i2c_soft_write_byte(self_p,byte,ACK);
+}
+
+/**
+ * Read a byte from I2C bus.
+ */
+static int read_byte(struct i2c_soft_driver_t *self_p,

```

Comment:
Remove ``start_cond()``, ``stop_cond()``, ``write_byte()`` and ``read_byte()`` and instead call the i2c driver functions directly.